Lib/test/test_os.py

```diff
@@ -1987,7 +1987,7 @@ def mock_execve(name, *args):
 
     try:
         orig_execv = os.execv
-        # os.execve not implemented yet for all platforms
+        # NOTE: os.execve not implemented yet for all platforms
```

Comment:
Is this in the CPython source?

Reply:
If it's not, we should change it to this:
```suggestion
        # TODO: RUSTPYTHON, os.execve not implemented yet for all platforms
```

Reply:
`execve` is implemented in `posix.rs`
Should I adjust comments for all not implemented `os.*` that are not implemented for windows?